src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,206 @@
+import {
+  FC, useEffect, useState,
+} from 'react';
+
+import { useMatch } from 'react-router-dom';
+
+import cn from 'classnames';
+
+import { Person } from '../../types';
+
+import { PersonLink } from '../PersonLink';
+
+import { Loader } from '../Loader';
+
+import { getPeople } from '../../api';
+
+export const PeopleTable: FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+
+  const [error, setError] = useState('');
+
+  const [isLoading, setIsLoading] = useState(false);
+
+  const match = useMatch('/people/:selectedSlug');
+
+  const [selectedSlug, setSelectedSlug] = useState(match?.params.selectedSlug);
+
+  const handlePersonClick = (slug: string) => {
+    setSelectedSlug(slug);
+  };
+
+  useEffect(() => {
+    setSelectedSlug(match?.params.selectedSlug || '');
+  }, [match]);
+
+  async function showPeople() {
+    try {
+      setIsLoading(true);
+      const peopleFromServer = await getPeople();
+
+      const updatedPeople = peopleFromServer.map(person => {
+        const mother = peopleFromServer.find((human) => (
+          person.motherName === human.name
+        ));
+
+        const father = peopleFromServer.find((human) => (
+          person.fatherName === human.name
+        ));
+
+        return {
+          ...person,
+          father,
+          mother,
+        };
+      });
+
+      if (updatedPeople.length === 0) {
+        setError('no people');
+      }
+
+      setPeople(updatedPeople);
+      setIsLoading(false);
+    } catch {
+      setError('something wrong');
+      setIsLoading(false);
+    }
+  }
+
+  useEffect(() => {
+    showPeople();
+  }, []);
+
+  return (
+    <>
+      {
+        isLoading
+          ? <Loader />
+          : (
+            <>
+              {
+                error
+                  ? (
+                    <p
```

Comment:
```suggestion
          : ( { error
                  ? (<p
```
There is also no need to add fragments everywhere (<>).
And try to make your code cleaner. Format it correctly